### PR TITLE
MMT-1034: Added Variables and Services

### PR DESCRIPTION
**Pull request by rdcody**
-Added Variables and Services controllers, paths, and views.
-Left Services commented out.
-Reorganization and styling will need to be done before Services is added because there isn't enough room in the header. 
-Fixed a problem with the modal window not being rounded because of Overflow: scroll.

### CODE REVIEW

File: spec/features/collection_drafts/collection_draft_deletion_spec.rb [hunk 1]
```diff
@@ -33,8 +33,13 @@
     #   end
     # end
 
+<<<<<<< HEAD
```
Comment (charleshuang80) [inline spec/features/collection_drafts/collection_draft_deletion_spec.rb]: rebase merge conflict stuff left in...

File: app/controllers/manage_collections_controller.rb [hunk 2]
```diff
@@ -1,12 +1,11 @@
 # :nodoc:
 class ManageCollectionsController < PagesController
   include BulkUpdates
-  include ManageCollectionsHelper
 
-  layout 'manage_collections'
+  layout 'manage_metadata'
```
Comment (charleshuang80) [inline app/controllers/manage_collections_controller.rb]: I thought we had discussed keeping a bare `ManageMetadataController` that the controllers for ManageCollections, ManageVariables, and ManageServices could all inherit from. Then we wouldn't need to have to declare this layout in the three different controllers. Had that decision been changed, or did those edits get lost somehow? I recall seeing that and discussing it on the PR review for 1033 the other day.

File: nohup.out [hunk 3]
```diff
@@ -0,0 +1 @@
+Error: Could not find or load main class cmr.dev_system.runner
```
Comment (charleshuang80) [inline nohup.out]: sneaked in again!

File: app/controllers/manage_collections_controller.rb (hunk 2 above)
Comment (rdcody) [inline app/controllers/manage_collections_controller.rb]: I think the plan was to keep the Manage Metadata controller empty, but also having a manage collections, variables and services controller. Although, we would only have one layout, the manage metadata layout, to accommodate all of their views. May need @macrouch or @abbottry to verify that, but that is what is in MMT-1033.

Comment (rdcody) [inline app/controllers/manage_collections_controller.rb]: Added back in the Manage Metadata Controller.

**Pull request merged**